.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,55 @@
+# Build and push the docs website
+name: Build docs
+
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+  schedule:
+    - cron: '0 */8 * * *'
```

Comment:
yeah this will be triggered every 8h and report in both the main readme and in the docs like https://github.com/ccamacho/topsail#topsail and https://ccamacho.github.io/topsail/intro.html#topsail